content/blog/2024/keynote-announcement-usha-rengaraju.md

```diff
@@ -0,0 +1,19 @@
+Title: Introducing our next Keynote Speaker for PyCon India 2024 - Usha Rengaraju
+Date: 2024-08-10 17:00
+Category: 2024
+Slug: keynote-announcement-usha-rengaraju
+Summary: We are excited to announce **Usha Rengaraju** as a Keynote Speaker for PyCon India 2024!
+
+*We know you've been waiting for this! We're thrilled to announce our next Keynote Speaker for PyCon India 2024!*
+
+<p align="center" data-aos="fade-right"  data-aos-duration="1000">
+    <img src="{static}/images/2024/keynote-usha-rengaraju.jpeg" alt="" class="img-fluid" style="border-radius: 10%; max-height: 200px;">
+</p>
+
+Usha is an AI consultant and corporate trainer and she is the World’s first women triple Kaggle Grandmaster .She specializes in Deep Learning , Generative AI , TigerGraph and Probabilistic Graphical Models and was also one of the judges of Tiger Graph “ Graph for all - Million Dollar Challenge” in 2022 and the judge for Intel oneAPI Hackathon organized by Intel software in 2022 .
+
+She is ranked as top ten Data Scientists in India for the year 2020 by Analytics India Magazine , ranked as top ten women data scientists by Analytics Insight magazine for 2021 and also ranked as top 150 AI Leaders and Influencers by 3AI magazine.
+
+She organized NeuroAI which is India’s first-ever research symposium in the interface of Neuroscience and Data Science. She also organized the Neurodiversity India Summit which is India’s first ever conference in Neurodiversity .
+
+She is one of the winners of “ML in Action” competition organized by the ML Developer programs team at Google and her team won first place in WiDS 2022 Datathon organized by Stanford University . She is also the winner of Kaggle ML Research spotlight for 2022 and winner of TensorFlow Community Spotlight 2023 .
```

Comment:
Usha is an AI consultant and a corporate trainer who specialises in Deep Learning, Generative AI, TigerGraph and Probabilistic Graphical Models.

However, that's the boring part, because... she is the World’s first woman triple Kaggle Grandmaster!

While it is hard to count her accomplishments over the years, we'd like to highlight a few just so that we know how lucky we are to have her as a keynote this year.

Her extensive knowledge got her invited as one of the judges for both Tiger Graph “Graph for all - Million Dollar Challenge” as well as the Intel oneAPI Hackathon organized by Intel software back in 2022.

As they say, one's work speaks for themselves and as such, hers was acknowledged by both the Analytics India magazine as well as the Analytics Insight Magazine when she was listed as one among the top 10 Data Scientists in India for the years 2020 & 2021 respectively. Also ranked as one of the top 150 AI Leaders and Influencers by the 3AI magazine, the dedication she has to her craft is truly awe inspiring.

Getting invited for events as well being acknowledged for their work is one thing, but passing on one's knowledge and being a part of grooming the next generation is only possible by one who holds the community around their craft close to their hearts. Our admiration for her work is only multiplied seeing how she took the time to organize NeuroAI - India’s first-ever research symposium in the interface of Neuroscience and Data Science! Along with that, the Neurodiversity India Summit as well, which is India’s first ever conference in Neurodiversity .

Yes, we've covered a few of her accomplishments. However, while a lot of people at this point in their lives usually feel as they've "done it", Usha is not one of them. As much as she is able to judge, she doesn't shy away from being judged, and hence keeping her competitive spirit alive making sure she is always at the top of her game:
- One of the winners of "ML in Action" competition organized by the ML Developer programs team at Google
- Her team won first place in WiDS 2022 Datathon organized by Stanford University
- The winner of Kaggle ML Research spotlight for 2022
- The winner of TensorFlow Community Spotlight 2023

Phew! Talk about being inspired! We can't wait ourselves to hear from her at PyCon India 2024. However, we are even more happy to have her at the conference for all of you to interact with and learn from.